### PR TITLE
Updated deps and moved cactoos to test scope

### PR DESCRIPTION
**Pull request by olenagerasimova**
Updated deps, used new `Inspector` multipart api and moved cactoos to test scope.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -152,13 +163,13 @@ SOFTWARE.
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-params</artifactId>
-      <version>${junit-platform.version}</version>
+      <version>5.7.1</version>
```
Comment (genryxy) [inline pom.xml:166]: why version was downgraded?

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one question

**Comment by olenagerasimova** [comment]
@genryxy thanks, upgraded version

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**